
gcc/testsuite/ChangeLog

    * gcc.dg/builtin-bswap-5.c: improve test vector to avoid nibble swaps passing

---

diff --git a/gcc/testsuite/gcc.dg/builtin-bswap-5.c b/gcc/testsuite/gcc.dg/builtin-bswap-5.c
index b29931e4e10..2ba6c31194f 100644
--- a/gcc/testsuite/gcc.dg/builtin-bswap-5.c
+++ b/gcc/testsuite/gcc.dg/builtin-bswap-5.c
@@ -6,13 +6,13 @@ main (void)
   /* Test constant folding.  */
   extern void link_error (void);
 
-  if (__builtin_bswap16(0xaabb) != 0xbbaa)
+  if (__builtin_bswap16(0xabcd) != 0xcdab)
     link_error ();
 
-  if (__builtin_bswap32(0xaabbccdd) != 0xddccbbaa)
+  if (__builtin_bswap32(0x89abcdef) != 0xefcdab89)
     link_error ();
 
-  if (__builtin_bswap64(0x1122334455667788ULL) != 0x8877665544332211ULL)
+  if (__builtin_bswap64(0x0123456789abcdefULL) != 0xefcdab8967452301ULL)
     link_error ();
 
   return 0;
